django_elasticsearch_dsl/fields.py

```diff
@@ -124,7 +124,7 @@ def get_value_from_instance(self, instance, field_value_to_ignore=None):
 
         if objs is None:
             return {}
-        if isinstance(objs, collections.Iterable):
+        if not isinstance(objs, SimpleLazyObject) and isinstance(objs, collections.Iterable):
```

Comment:
This should be better approach as the `SimpleLazyObject` can also be a `Iterable`.
```suggestion
        try:
            is_iterable = bool(iter(objs))
        except TypeError:
            is_iterable = False
        
        if is_iterable:
```